modules/Hoot/ui/managePanel/jobs.js

```diff
@@ -960,10 +960,10 @@ export default class Jobs extends Tab {
                         }
                     });
                     actions.push({
-                        title: 'open in josm',
+                        title: 'open in JOSM',
```

Comment:
the tooltip convention I have going in the jobs panel is for them to be all lowercase, so this wasn't a typo